src/db.py

```diff
@@ -105,6 +113,14 @@ def add_user(self, user: dict):
         else:
             return STATUS_FAIL
 
+    def get_user(self, user_id: int):
+        session = self._sm()
+        user = session.query(User).filter(User.tg_id == user_id).one()
+        if user:
+            return user.tg_id
+        else:
+            return None
```

Comment:
Seems a bit strange. The function should answer the question "is user already exists?". Consequently:
- lets rename it (for instance, is_user_exist() ). Current name is confusing. If it's "get_user", it should return a user. But it returns "user_id" we already know or None.
- the function should return boolean value True/False